CONTRIBUTING.md

```diff
@@ -24,6 +24,8 @@ conda config --add channels conda-forge
 
 Once you have conda ready:
 
+Fork this repository to your own GitHub account.
```

Comment:
There's a command you can specify here that forks before the git clone

Reply:
maybe a screenshot is better? (showing the fork button) i'd bet not many people use `gh`'s CLI

Reply:
Added a screenshot of the fork button.